src/Model/RequestManager.php

```diff
@@ -0,0 +1,43 @@
+<?php
+/**
+ * Created by WCS.
+ * User: Kevin
+ * Date : 28/10/2020
+ */
+
+namespace App\Model;
+
+class RequestManager extends AbstractManager
+{
+    const TABLE = 'request';
+    const PROBLEM = 'You have a problem';
```

Comment:
Constant not used, should probably be removed

Reply:
ok, deleted.